libraries/lib-datahandler/src/main/thrift/licenseinfo.thrift

```diff
@@ -56,6 +69,20 @@ service LicenseInfoService {
 
     /**
      * get a copyright and license information file on all linked releases and linked releases of linked projects (recursively)
+     * output format as specified by outputType
      */
-    string getLicenseInfoFileForProject(1:  string projectId, 2: User user);
+    string getLicenseInfoFileForProject(1: string projectId, 2: User user, 3: string outputType);
+
+    binary getLicenseInfoFileForProjectAsBinary(1: string projectId, 2: User user, 3: string outputType);
+
+    /**
+      * returns all available output types
+      */
+    list<OutputFormatInfo> getPossibleOutputFormats();
+
+    /**
+     * returns file extension that is applicable when licenseInfo is generated by the generator class
```

Comment:
doc comment isn't quite correct. the method returns more than just the file extension.